qiskit/providers/aer/quantum_info/states/aer_state.py

```diff
@@ -0,0 +1,297 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019, 2020, 2021, 2022
```

Comment:
It doesn't really matter but I'd just put:

```suggestion
# (C) Copyright IBM 2022
```

You could also just do a date range if you though the full date was needed